var/spack/repos/builtin/packages/py-libsonata/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyLibsonata(PythonPackage):
+    """SONATA files reader"""
+
+    homepage = "https://github.com/BlueBrain/libsonata"
+    git = "https://github.com/BlueBrain/libsonata.git"
+    pypi = "libsonata/libsonata-0.1.14.tar.gz"
+
+    submodules = True
+
+    maintainers("tristan0x")
+
+    version("develop", branch="master", get_full_repo=True)
```

Comment:
Is there a reason the name of the version doesn't match that of the branch?  

Normally they are the same.  For more on version comparisons, see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison

Reply:
Hello @tldahlgren It thought all development spack versions had to be named `develop`. I updated it to "master"  in 2de655c6743db645ab5621ab705e4f1b49651168